.github/renovate.json

```diff
@@ -12,5 +12,8 @@
     "commitMessageExtra": "from {{currentVersion}} to {{#if isMajor}}v{{{newMajor}}}{{else}}{{#if isSingleVersion}}v{{{toVersion}}}{{else}}{{{newValue}}}{{/if}}{{/if}}",
     "commitMessageSuffix": "({{packageFile}})",
     "labels": ["dependencies"]
-  }]
+  }],
```

Comment:
Then basically this is not needed according to https://docs.renovatebot.com/modules/manager/helm-requirements/#additional-information

Reply:
Yes, but without it, renovate does not update the dependencies. This is the relevant log from renovate:

```
         "helmv3": [
           {
             "deps": [
               {
                 "depName": "mysql",
                 "currentValue": "~9.1.7",
                 "skipReason": "placeholder-url",
                 "updates": [],
                 "packageName": "mysql"
               },
               {
                 "depName": "postgresql",
                 "currentValue": "~11.6.5",
                 "skipReason": "placeholder-url",
                 "updates": [],
                 "packageName": "postgresql"
               },
               {
                 "depName": "postgresql-ha",
                 "currentValue": "~9.1.5",
                 "skipReason": "placeholder-url",
                 "updates": [],
                 "packageName": "postgresql-ha"
               },
               {
                 "depName": "rabbitmq",
                 "currentValue": "~11.2.0",
                 "skipReason": "placeholder-url",
                 "updates": [],
                 "packageName": "rabbitmq"
               },
               {
                 "depName": "redis",
                 "currentValue": "~16.12.0",
                 "skipReason": "placeholder-url",
                 "updates": [],
                 "packageName": "redis"
               }
             ],
             "datasource": "helm",
             "packageFileVersion": "1.6.109-dev",
             "lockFiles": ["helm/defectdojo/Chart.lock"],
             "packageFile": "helm/defectdojo/Chart.yaml"
           }

```

With the setting, it looks like this (excerpt):

```
           "branchName": "renovate/mysql-9.x",
           "prNo": null,
           "prTitle": "Update Helm release mysql from 9.1.8 to ~9.19.0 (helm/defectdojo/Chart.yaml)",
           "result": "branch-limit-reached",
           "upgrades": [
             {
               "datasource": "helm",
               "depName": "mysql",
               "displayPending": "",
               "currentVersion": "9.1.8",
               "currentValue": "~9.1.7",
               "newValue": "~9.19.0",
               "newVersion": "9.19.1",
               "packageFile": "helm/defectdojo/Chart.yaml",
               "updateType": "minor",
               "packageName": "mysql"
             }
           ]
         },

```

Reply:
Yea, if I'm able to interpret the code correctly, looks like if the alias is not defined, it will  throw `placeholder-url`
https://github.com/renovatebot/renovate/blob/ea5e20ebb1fd7108a70800514b28eba92c27f3a5/lib/modules/manager/helmv3/extract.ts#L68 ;)

So let's merge it